### ISSUE TITLE
Replace Layer

### ISSUE THREAD

**Issue by christophersybico** [getflourish/Sketch-Mate#11]
I think the Replace Layer (⌘ + ⌥ + R) stopped working on Sketch 3.4.

**Comment by gregwburke** [comment]
Yup. Broken for me too. Indispensable plugin for me these days, so this is a kick in the nuts.

**Comment by ohcommon** [comment]
Guys, I fixed the issue, but I'm not comfortable with git yet to send it like pull request.

It seems like Replace Layer doesn't work now because it relies on `Edit › Paste in Place` command which is missing in Sketch 3.4. It uses `Edit › Paste Over Selection` instead.

In order to fix it you need to do the following:

1) In "inventory.js" add:

``` javascript
sendPasteOverSelection: function () {
  my.utils.sendAction('pasteOverSelection:');
},
```

just after these lines:

``` javascript
sendPasteInPlace: function () {
  my.utils.sendAction('pasteInPlace:');
},
```

2) In Replace Layer.sketchpluginand change 21st line from 

``` javascript
com.getflourish.utils.sendPasteInPlace();
```

to

``` javascript
com.getflourish.utils.sendPasteOverSelection();
```

I hope this helps.

**Comment by ohcommon** [comment]
Just noticed that @girafic came up with more elegant solution earlier and it's already in pull requests. My bad.

**Comment by megaroeny** [comment]
@girafic thank you! I so need this back :+1:

**Comment by christophersybico** [comment]
awesome thanks!

**Comment by gregwburke** [comment]
Can't wait to get this merged!

**Comment by BorisBozic** [comment]
+1 for merging the pull request to fix this (hopefully). Didn't realise how much I used this shortcut until I couldn't. Thanks for all your work on this.

**Comment by earldigger** [comment]
+1 for merging, wow, I can't believe how much I use this. Thanks for all the work, it has become indispensable for me.

**Comment by RaphyRaph** [comment]
Meeeeeeeeeeeeeerge plize!

**Comment by getflourish** [comment]
Omg didn’t see this until now. Thanks @ohcommon for the proposal. Fixed in e42f3301eeeda2b3dd50535692f7f8287d072ab1
